src/providers/cortex/trulens/providers/cortex/provider.py

```diff
@@ -2,12 +2,18 @@
 import os
 from typing import ClassVar, Dict, Optional, Sequence
 
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from dotenv import load_dotenv
 import snowflake
 import snowflake.connector
 from snowflake.connector import SnowflakeConnection
+import streamlit as st
```

Comment:
I feel like we shouldn't depend on streamlit or python-dotenv here for passing connection details to a provider. Streamlit is a UI library and there is no indication that a .env is being read. Can we have __init__ accept account, user, role, etc as arguments instead?

Reply:
Updated